### PR TITLE
Adding myself to the CI team

### PR DESCRIPTION
**Pull request by karthikv2k**
I am the lead developer for Kubeflow/Fairing and work on integration tests for it so need access to the project for debugging and setup.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/internal-acls/101)

### CODE REVIEW

File: ci-team.members.txt [hunk 1]
```diff
@@ -12,3 +12,4 @@ michellecasbon@google.com
 redhat-team@kubeflow.org
 wibuch@microsoft.com
 yangpa@google.com
+caritck@google.com
```
Comment (abhi-g) [inline ci-team.members.txt]: move alphabetically

File: google-team.members.txt [hunk 2]
```diff
@@ -13,3 +13,4 @@ thealamkin@google.com
 yxshi@google.com
 zhenghui@google.com
 zahraabbasi@google.com
+cartick@google.com
```
Comment (abhi-g) [inline google-team.members.txt]: move up alphabetically

Comment (karthikv2k) [reply google-team.members.txt]: done

**Pull request merged**